app/services/doc_auth/selfie_concern.rb

```diff
@@ -0,0 +1,23 @@
+module DocAuth
+  module SelfieConcern
+    extend ActiveSupport::Concern
+    def selfie_live?
+      portrait_match_results ||= {}
```

Comment:
This line is because the Acuant result doesn't actually have the portrait match. Feels like there should be a better way to handle this?

Reply:
That was my original idea, but doing that doesn't work because the acuant result doesn't have the portait_match_results:
- Remove the `portrait_match_results ||= {}` lines
- Run: `bundle exec rspec spec/services/doc_auth/acuant/responses/get_results_response_spec.rb`

Reply:
Then we may try  dynamically define these method in the concern with 
```ruby
include do

end
```

Reply:
Seems like we're going to use [this](https://github.com/18F/identity-idp/pull/9941#discussion_r1463623736) approach for this.